helix-term/src/ui/prompt.rs

```diff
@@ -355,15 +360,36 @@ impl Prompt {
             }
         }
 
-        if let Some(doc) = (self.doc_fn)(&self.line) {
-            let mut text = ui::Text::new(doc.to_string());
+        if let Some(PromptDoc { doc, aliases }) = (self.doc_fn)(&self.line) {
+            let mut string = String::with_capacity(
+                doc.len() + aliases.iter().fold(0, |acc, alias| acc + alias.len() + 2) + 12,
+            );
+            string.push_str(doc);
+
+            if !aliases.is_empty() {
+                string.push('\n');
+                string.push_str("Aliases: ");
```

Comment:
Shouldn't we show `Alias` if there is an alias and `Aliases` if there are multiple aliases?

Reply:
You're right, there are some changes I want to make to this section of the code, thank you for the reminder! I'll see if I can work on it today.

Reply:
I think it's okay to use `Aliases:` it's one or more aliases

Reply:
Just as an additional point in favor of "it doesn't really matter" (IMHO): Kakoune displays `Aliases:` even when there's only one alias:

![image](https://user-images.githubusercontent.com/28582702/140866039-15f6b3fd-dbf5-47ff-91f3-d9650af6b3b2.png)